core/designsystem/src/main/java/com/mifos/core/designsystem/component/MifosScaffold.kt

```diff
@@ -73,5 +73,21 @@ fun MifosScaffold(
     ) { padding ->
         content(padding)
     }
+}
+
+@Composable
+fun MifosScaffoldNoTopBar(
```

Comment:
Expose the Topbar in the above function and make it nullable and then update this function by passing null for topbar.

Reply:
I removed this composable and made the changes that you suggested.